lib/features/booking/add_inspection_success_booking_screen.dart

```diff
@@ -71,7 +71,7 @@ class AddInspectionSuccessScreen extends StatelessWidget {
                   price: 4800,
                   state: 'Vitoria',
                   currency: Currency.aud,
-                  rentingPeriod: 'pw',
+                  paymentPeriod: HatSpaceStrings.current.pw,
```

Comment:
Use "pm" here, as our price is always per month. I left a comment on figma

Reply:
already fixed it with 'pm'